upcomingevents.json

```diff
@@ -42,6 +42,13 @@
       "desc": "A competition hosted by Stanford (hopefully online) where teams of 1-3 students participate in 2 rounds of programming problems.",
       "tentative": true
     },
+    {
+      "name": "Noctem ABCIC 2020",
+      "link": "https://noctem.dev/ABCIC2020.html",
+      "time": 1605409200,
+      "desc": "A competition hosted by the Noctem development team(Hopefully Online) where teams of 1-5 students work together to solve problems ranging from USACO Bronze to Gold difficulty. The date is not finalized, but it is in Fall 2020",
```

Comment:
don't capitalize hopefully online, also you can remove the "date is not finalized" part, just say sometime during fall (it's a tentative date, it'll be indicated).